testdata/project-v4-multigroup-with-deploy-image/api/v1/lakers_webhook.go

```diff
@@ -31,45 +32,48 @@ var lakerslog = logf.Log.WithName("lakers-resource")
 func (r *Lakers) SetupWebhookWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewWebhookManagedBy(mgr).
 		For(r).
+		WithValidator(r).
+		WithDefaulter(r).
 		Complete()
 }
 
 // TODO(user): EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 
 //+kubebuilder:webhook:path=/mutate-testproject-org-v1-lakers,mutating=true,failurePolicy=fail,sideEffects=None,groups=testproject.org,resources=lakers,verbs=create;update,versions=v1,name=mlakers.kb.io,admissionReviewVersions=v1
 
-var _ webhook.Defaulter = &Lakers{}
+var _ admission.CustomDefaulter = &Lakers{}
 
-// Default implements webhook.Defaulter so a webhook will be registered for the type
-func (r *Lakers) Default() {
+// Default implements admission.CustomDefaulter so a webhook will be registered for the type
+func (r *Lakers) Default(ctx context.Context, obj runtime.Object) error {
 	lakerslog.Info("default", "name", r.Name)
 
 	// TODO(user): fill in your defaulting logic.
+	return nil
 }
 
 // TODO(user): change verbs to "verbs=create;update;delete" if you want to enable deletion validation.
 //+kubebuilder:webhook:path=/validate-testproject-org-v1-lakers,mutating=false,failurePolicy=fail,sideEffects=None,groups=testproject.org,resources=lakers,verbs=create;update,versions=v1,name=vlakers.kb.io,admissionReviewVersions=v1
 
-var _ webhook.Validator = &Lakers{}
+var _ admission.CustomValidator = &Lakers{}
```

Comment:
Should we really use Lakers{} Or should we create a new struct for the validator?

c/c @nathanperkins ^